src/components/Header/Header.tsx

```diff
@@ -1,15 +1,38 @@
-import React from 'react';
+import React, { useState } from 'react';
+// import { Switch, makeStyles, ThemeProvider, createTheme } from '@material-ui/core';
+// import FormGroup from '@material-ui/core/FormGroup';
+// import { alpha, styled } from '@material-ui/core/styles';
+import Switch from 'react-switch';
 import { HeaderProps } from '../../interfaces/interfaces';
```

Comment:
you don't need to extract HeaderProps to the global interfaces. U use it only on a component level. It's ok to keep it in the same file